### DISCUSSION TITLE
Email digest of Github Community?

### DISCUSSION THREAD

**Discussion by j-peeters** [beetbox/beets#3981] [Q&A]
Hi, does anyone know if there is a setting to receive a weekly email digest for all activity in the Beets Github Community? I don't seem to find it. Thanks, Jan

**Comment by jef** [comment]
Great question. I don't think there is anything like that here. The closest thing to that is probably using [Notifications](https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications).

That's a nice thing with Discourse. Since GitHub Discussions is still in Beta, I bet there will be something there soon enough.

**Reply by j-peeters** [reply]
Good to know Jef, thanks. 
I'll file a feature request with GitHub. Hopefully they implement it.

**Reply by jef** [reply]
Great idea. I think it'd be a good addition in general.
